gensim/models/word2vec.py

```diff
@@ -39,18 +39,15 @@
 
 .. sourcecode:: pycon
 
-    >>> from gensim.test.utils import common_texts, get_tmpfile
+    >>> from gensim.test.utils import common_texts
     >>> from gensim.models import Word2Vec
     >>>
-    >>> path = get_tmpfile("word2vec.model")
-    >>>
     >>> model = Word2Vec(common_texts, size=100, window=5, min_count=1, workers=4)
     >>> model.save("word2vec.model")
 
-The training is streamed, meaning `sentences` can be a generator, reading input data
-from disk on-the-fly, without loading the entire corpus into RAM.
+`common_texts` is a list of lists of strings, where each list is a document.
```

Comment:
This is not good – the streaming is essential. We've had many instances where people read the docs and assume the input **must** be a list, which is not the case.

Please keep the streaming (iterable) language in, just drop the generator.

Reply:
Ah OK, I'll fix that.